### ISSUE TITLE
fix/check grRules for Oxphos reactions in iSM996

### ISSUE THREAD

**Issue by IVANDOMENZAIN** [SysBioChalmers/ecModels#30]

**Comment by IVANDOMENZAIN** [comment]
@simas232 This is something to think about, regarding the representation of OxPhos in `iSM996`.

**Comment by mihai-sysbio** [comment]
It sounds to me like this problem would be better solved [upstream](https://github.com/SysBioChalmers/Kluyveromyces_marxianus-GEM). `ecModels` could then, at least for a while, use a specific branch of the source repository with a direct link to the `.xml`.

**Comment by IVANDOMENZAIN** [comment]
> It sounds to me like this problem would be better solved [upstream](https://github.com/SysBioChalmers/Kluyveromyces_marxianus-GEM). `ecModels` could then, at least for a while, use a specific branch of the source repository with a direct link to the `.xml`.

I agree @mihai-sysbio, sounds like the solution should be in the original repo. To accelerate the process I can replicate this issue in the original GEM repo and create a new PR with my suggested solution (already implemented locally).
